geoportailv3/templates/index.html

```diff
@@ -252,16 +252,14 @@ <h2 translate>Feedback</h2>
         </div>
         <div ng-if="mainCtrl.selectedLayers.length>0" class="slider-layer-label"><i class="fa fa-arrow-left"></i> <span>{{mainCtrl.selectedLayers[0].get('label') | translate}}</span></div>
       </div>
-      <ngeo-offline class="offline-debug" ng-if="::mainCtrl.debugOffline"
-          debug="::true"
+      <ngeo-offline class="offline-debug" ng-class="{'offline-debug': mainCtrl.debugOffline}"
```

Comment:
You must remove the class (already in your ng-class)

Reply:
You probably can use `::mainCtrl.debugOffline` in your ng-class (with `::`)

Reply:
We can also get rid of the offline-debug stuff now.

Reply:
FYI `{'offline-debug': ::mainCtrl.debugOffline}` is not a valid syntax.

I'll remove the stuff about debug then.